.github/workflows/ci-scrypto-builder.yml

```diff
@@ -1,4 +1,4 @@
-name: Simulator
+name: scrypto builder ci
```

Comment:
What's the purpose of this workflow out of interest? To show the build works? 

I'm not sure we necessarily need this if it'll be expensive to run on ever PR/Push

But happy if you want it

Reply:
hmm good point. what if we run on every merge to develop/main? the purpose is just to make sure that the docker image can still be built and it can be used to update the assets

Reply:
we can also build if and only if there were changes to the assets or the simulator folder